navi/src/test/java/com/trello/navi/NaviFragmentTest.java

```diff
@@ -295,4 +295,9 @@ public void restoreInstanceStatePersistableListener() {
     exception.expect(IllegalArgumentException.class);
     emitter.addListener(Event.DETACHED_FROM_WINDOW, mock(Listener.class));
   }
+
+  @Test public void onViewCreatedListener() {
```

Comment:
This test should be verifying that you *can* use this event, not that it throws an exception.